### ISSUE TITLE
[feature request] Sort, Deduplicate, lowercase, Title Case, etc.

### ISSUE THREAD

**Issue by meoso** [richrd/suplemon#210]
Would sorting a highlighted area be possible, or not possible due to no region support?

**Comment by richrd** [comment]
Actually sorting lines is comming in the next release. It will allow sorting alphabetically or by length, and reversing the sort order. Lower case (and upper case) are available now via commands `lower` and `upper`. Run them with `Ctrl + E`. All these are line based, so they effect all lines that have cursors on them.

Deduplicate and title case aren't available, but should be easy to implement. I assume deduplicate removes all duplicates from the selected lines?

**Comment by meoso** [comment]
Yes to last question. TYVM; suplemon is fantastic.  I cannot locate a list of available commands. +1>Wiki

**Comment by richrd** [comment]
Thanks, and good point!

Luckily I just added the command list. It'll be included in the readme and help feature of the next version (0.1.64) which is pretty much ready for release. You can see it here (scroll all the way down to 'Commands') https://github.com/richrd/suplemon/blob/dev/README.md

Also the run command prompt has auto-complete so you can cycle through all commands by hitting <kbd>tab</kbd>.
